src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  error: string;
+  onCloseError: () => void;
+};
+
+export const ErrorMessage: React.FC<Props> = ({ error, onCloseError }) => {
+  const handleCloseError = () => {
+    onCloseError();
+  };
```

Comment:
you need to create setTimeout on the first render and clear timeout on unmount, use useEffect

Reply:
Fixed. I also added a trigger to avoid firing useEffect on the clear error value